### PR TITLE
Drop support for PHP7.4, Symfony version upgraded to 5.4

### PR DESCRIPTION
**Pull request by Rafikooo**
| Q               | A
| --------------- | -----
| Branch?         | master 
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no 
| License         | MIT

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -17,19 +17,13 @@ jobs:
...
-                    -
-                        php: 7.4
-                        symfony-version: 4.4.*
-                    -
-                        php: 7.4
-                        symfony-version: 5.1.*
                     -
                         php: 8.0
                         symfony-version: 4.4.*
                         composer-flags: "--ignore-platform-req=php"
```
Comment (lchrusciel) [inline .github/workflows/build.yml:23]: can we drop this flag?

Comment (Rafikooo) [reply .github/workflows/build.yml]: Should we drop only this flag or in both symfony-version: 4.4.* and symfony-version: 5.4.* configs?

**Review by GSadee — APPROVED** [review]

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -17,19 +17,12 @@ jobs:
...
-                        symfony-version: 5.1.*
                     -
                         php: 8.0
                         symfony-version: 4.4.*
-                        composer-flags: "--ignore-platform-req=php"
                     -
                         php: 8.0
-                        symfony-version: 5.1.*
+                        symfony-version: 5.4.*
                         composer-flags: "--ignore-platform-req=php"
```
Comment (loic425) [inline .github/workflows/build.yml:26]: We can also drop this composer flags here.

**Review by loic425 — DISMISSED** [review]

**Comment by GSadee** [comment]
Thanks, Rafał! :tada:

**Pull request merged**